Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -82,4 +189,171 @@ - (NSArray*)getListOfContacts {
     return result;
 }
 
+- (NSArray*)getListOfModifiableContacts {
```

Comment:
>getListOfModifiableContacts [](start = 12, length = 27)

These two functions look ripe for turning into a helper that turns a contactList into a list of _ABContacts